lochness/mediaflux/__init__.py

```diff
@@ -127,19 +128,25 @@ def sync_module(Lochness: 'lochness.config',
 
                             # construct local path
                             protect = prod.get('protect', False)
+                            processed =  prod.get('processed', False)
                             key = enc_key if protect else None
                             subj_dir = subject.protected_folder \
                                 if protect else subject.general_folder
 
-                            mf_local= pjoin(subj_dir, datatype, dirname(patt), basename(remote))
+                            # mf_local= pjoin(subj_dir, datatype, dirname(patt), basename(remote))
```

Comment:
@tashrifbillah I've replaced this line to utilize `tree.get` module, to centralize the output structure function. Also now receiving `processed` boolean from the `config.yml`. To repeat, we are not using the `processed` in the Pronet and Prescient, but keeping the option for future developments and other projects.